### DISCUSSION TITLE
Limit errors?

### DISCUSSION THREAD

**Discussion by jessekrubin** [samchon/typia#1310] [Q&A]
Is there a way to limit the number of errors generated during validation so that my thingy doesn’t have to traverse the whole item being validated and/or so my thingy doesn’t end up generating an ungodly number of error objects?

**Comment by samchon** [comment]
No way.

**Reply by jessekrubin** [reply]
> No way

as in "there is no way to do that?"

or as in "no way! I can't believe @jessekrubin is hitting me up?"


Assuming the former, how involved do you think such a feature would be? For a complex object I can get lot's o' error objects. Typebox yields errors as a generator which is quite handy.

**Comment by samchon** [comment]
Closing as answered by the maintainer: `typia.validate<T>()` does not have a `maxErrors` or error-limit option.

If you only need a boolean result, use `typia.is<T>()`. If you need validation error details, `validate()` generates the full error list; slicing `result.errors` afterward can limit what you display, but it does not reduce the validation traversal.
